versions/3.0.md

```diff
@@ -66,8 +66,8 @@ An OpenAPI description can then be used by documentation generation tools to dis
 
 ## Definitions
 
-##### <a name="oasDescription"></a>OpenAPI Description
-A document or set of documents that describe an API. An OpenAPI description uses and conforms to the OpenAPI Specification.
+##### <a name="oasDocument"></a>OpenAPI Document
+A document or set of files that document an API. An OpenAPI document uses and conforms to the OpenAPI Specification.
```

Comment:
We seem to have gone backwards slightly here, where we removes 'files' before. 

> A document or set of documents that

or

> A document (or set of documents) that